portal/templates/layouts/courseMan.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+
+<header>
+  <h2>Course Management</h2>
+  <a href="{{ url_for('index')}}">Home</a>
```

Comment:
this isn't wrong, but it would probably be better to put this on a nav bar on the base template or something, so that you have a home button on every page.  But we haven't really discussed what to put on, or where to put a nav bar yet, so don't worry about it